src/main.rs

```diff
@@ -98,6 +105,120 @@ enum CallOrHash {
 	Hash([u8; 32]),
 }
 
+// All the info associated with a call in the forms you may need it in.
+struct CallInfo {
+	call: NetworkRuntimeCall,
```

Comment:
This isn't actually handed out or used anywhere so a small cahnge would be to use `Network` here instead.